src/config/dynamic-theme-fixes.config

```diff
@@ -1141,6 +1141,16 @@ IGNORE INLINE STYLE
 
 ================================
 
+fanatical.com
+
+INVERT
+div.ksp:nth-child(2) > div:nth-child(1) > a:nth-child(1) > img:nth-child(1)
```

Comment:
`.trustpilot > .logo-container > a > img` Should do the trick if they change the DOM order.

Reply:
ok.